src/components/vanilla/charts/DynamicAxisBar/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEmbeddableState } from '@embeddable.com/react';
+import React, { useState, useEffect, useRef } from 'react';
```

Comment:
`useEffect` and  `useRef` are not used in the code

Reply:
@kuka-radovan I removed useRef - useEffect is used.